README.md

```diff
@@ -250,13 +250,13 @@ The Author agent (which is configured as multi-tenant) exposes its Admin API on
 Open a second bash shell (cd to the directory where you have checked out this repository) and run the BDD tests with:
 
 ```bash
-LEDGER_URL=http://localhost:9000 TAILS_SERVER_URL=http://localhost:6543 ./manage run-bdd
+LEDGER_URL=http://localhost:9000 TAILS_SERVER_URL=http://localhost:6543 poetry run ./manage run-bdd
```

Comment:
Why is `poetry run` needed here? The `manage` script should be able to be executed without this dependency.

Reply:
the `behave` executable is missing. We could move the `poetry run` into the manage script it's self.